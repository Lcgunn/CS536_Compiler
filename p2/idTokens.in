Hell0
H_3ll_oOo
123_a4
void