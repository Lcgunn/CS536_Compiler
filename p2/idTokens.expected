Hell0
H_3ll_oOo
!!! UNKNOWN TOKEN !!!
_a4
!!! UNKNOWN TOKEN !!!
